src/index.html

```diff
@@ -11,6 +11,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big">
+      <div class="search-big">
+        <label for="big">
+          <input id="big"
+                 class="bigest"
+                 type="search"
+                 data-qa="keypress"
+                 placeholder="Try &#34;Los Angeles&#34;">
```

Comment:
Неправильне форматування
```suggestion
          <input 
            id="big"
            class="bigest"
            type="search"
            data-qa="keypress"
            placeholder="Try &#34;Los Angeles&#34;"
          >
```